lib/peerflags/peerflags.go

```diff
@@ -109,17 +109,23 @@ func GetConfig(v *viper.Viper, repoPathEnv, defaultRepoPath string, isAuctioneer
 	if repoPath == "" {
 		repoPath = defaultRepoPath
 	}
+
+	connMan, err := connmgr.NewConnManager(
+		v.GetInt("conn-low"),
+		v.GetInt("conn-high"),
+		connmgr.WithGracePeriod(v.GetDuration("conn-grace")),
+	)
+	if err != nil {
+		return peer.Config{}, fmt.Errorf("creating conn manager: %s", err)
+	}
```

Comment:
`connmgr` dependency update had breaking APIs. Nothing weird, this is equivalent as before.